### DISCUSSION TITLE
Unable to get internationalization

### DISCUSSION THREAD

**Discussion by padeville** [Shopify/storefront-api-feedback#103] [Help]
It works with the rest api
It works in liquid 

but with the store front api 

`query getProductList($after: String) @inContext(language: ES) {
  products(first: 12, after: $after) {
    edges {
      node {
        id
        handle
        title`

I always get my products in English 

I tried to do it using the hydrogen app too but every time I open the language selector I get a 500 maybe it's link ?

**Comment by fahjan** [comment]
same issue here
